### PR TITLE
Minor Bug Fixes

### PR DESCRIPTION
**Pull request by hashb**
- cleared stale outputs
- Fixed a bug in `Keras Demo - Multiclass.ipynb`

Looks like someone forgot to convert validation set lablels

```python
# convert class vectors to binary class matrices
y_train = keras.utils.to_categorical(y_train, num_classes)
y_test  = keras.utils.to_categorical(y_test,  num_classes)
y_validation = keras.utils.to_categorical(y_validation, num_classes)
```

Also, I had some trouble with graphviz on macos. I was able to fix it by `brew install graphviz`.

### CODE REVIEW

**Comment by anniecherk** [comment]
Good catch, thanks!
